website/docs/docs/supported-data-platforms.md

```diff
@@ -3,81 +3,102 @@ title: "Supported Data Platforms"
 
-Community-supported plugins are works in progress, and anyone is welcome to contribute by testing and writing code. If you're interested in contributing:
-- Join both the dedicated #adapter-ecosystem channel in [dbt Slack](https://community.getdbt.com/) and the channel for your adapter's data store (e.g. #db-sqlserver, #db-athena) 
-- Check out the open issues in the plugin's source repository
+To learn more about adapters, check out ["What Are Adapters"](contributing/adapter-development/1-what-are-adapters).
+
+## Adapter Installation
+
+With two exceptions [^1], all adapters listed below can be installed from PyPI using `pip install <ADAPTER-NAME>`. The installation will include `dbt-core` and any other required dependencies, which may include other adapter plugins. Read more about [installing dbt](dbt-cli/install/overview).
+
+## Adapter Taxonomy
+
+### Verified by dbt Labs
+
+We have two tiers for dbt-core adapters: "Verified" and "Community".
+
+| Tier      | Relevant adapters                                                                                                                                                                                                                                                                                  |
+| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Verified  | The adapters: <li>maintained by dbt Labs as well as</li><li> partner-maintained adapters that have been verified by dbt Labs</li>                                                                                                                                                                  |
+| Community | <li>All other partner-maintained adapters as well as</li> <li>all community adapters.</li> Community adapters in particular tend to be works-in-progress. Contributions from users are highly encouraged (see ["Contributing to a pre-existing adapter"](#contributing-to-a-pre-existing-adapter)) |
+
+### Maintainers
+
+Who made and maintains and adapter is certainly relevant, but we recommend using an adapter's verification status to determine quality and health of an adapter. So far we have three categories of maintainers:
+
+| Supported by | Description                                                                                                                |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------- |
+| dbt Labs     | dbt Labs maintains a set of adapters for some of the most common databases, warehouses, and platforms.                     |
```

Comment:
> adapters

For consistency, should this be "adapter plugins" as is the case in the rest of the table? Also, I think this is a good opportunity to just be open about how we ended up with the ones we maintain. Something about citing that these adapter plugins were developed by dbt Labs in the earliest days of dbt Core and were initially managed together in the same repository. We no longer take on maintaining any new adapters as Partners are better suited to do this both from a scale and expertise standpoint (which is covered in the line below)

Reply:
```suggestion
| dbt Labs     | dbt Labs maintains a set of adapter plugins for some of the most common databases, warehouses, and platforms. As for why particular data platforms were chosen, see ["Why Verify an Adapter"](7-verifying-a-new-adapter#why-verify-an-adapter)                 |
```